packages/nextjs/app/page.tsx

```
@@ -16,7 +22,7 @@ const Home: NextPage = () => {
           <p className="text-center text-lg">Get started by taking a look at your batch GitHub repository.</p>
           <p className="text-lg flex gap-2 justify-center">
             <span className="font-bold">Checked in builders count:</span>
-            <span>To Be Implemented</span>
+            <span>{checkedInCount?.toString() || "0"}</span>
```

Maybe the fallback value should be different? Using 0 might be misleading, as it suggests no one has checked in yet.